classes/Category.php

```diff
@@ -690,7 +691,8 @@ public static function getAllCategoriesName(
         );
 
         if (!Cache::isStored($cacheId)) {
-            $result = Db::getInstance()->executeS('
+            $result = Db::getInstance()->executeS(
+                '
```

Comment:
Isn't weird to see an empty line? In this example I would expect the SQL request to start immediately after the quote instead of having a carriage return

Reply:
I can fix it, but please let me do it in an other PR. I already have an open PR related to SQL requests, and I do not want to start a lot of conflicts. :)